### DISCUSSION TITLE
Colorblind mode

### DISCUSSION THREAD

**Discussion by edrichhans** [community/community#48675] [Accessibility] [answered]
Hi, is it possible to add the colorblind themes on mobile?

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @edrichhans! Happy to have you here 🙂

If you have low vision, you may benefit from a high contrast theme, with greater contrast between foreground and background elements. Our light and dark themes may fit the bill there, and can be used on Mobile as well. You can learn more here: [Managing your theme settings](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-theme-settings).

**Comment by code-fanatic** [comment, marked as answer]
> Hi, is it possible to add the colorblind themes on mobile?
if you use an Android Device here is the solution:
- Open up the Settings.
- Tap System > Accessibility.
- Tap the Color correction option.
- Turn this feature on.
- Select from one of three color blind modes.
Deuteranomaly (red-green)
Tritanomaly (blue-yellow)
Protanomaly (red-green)

Tell me if this solved your problem.

**Reply by cicada1970** [reply]
Thanks for helping me out 😄

**Comment by cehfisher** [comment]
Hi @edrichhans - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/48675#discussioncomment-6608077). Could you please close this issue? Thank you! (cc @queenofcorgis)
